src/lv_interop.cc

```diff
@@ -154,9 +162,17 @@ namespace grpc_labview
     void SetLVString(LStrHandle* lvString, std::string str)
     {
         auto length = str.length();
-        auto error = NumericArrayResize(0x01, 1, lvString, length);
-        memcpy((**lvString)->str, str.c_str(), length);
-        (**lvString)->cnt = (int)length;
+
+        LStrHandle utf8String = nullptr;
+        auto error = NumericArrayResize(0x01, 1, &utf8String, length);
+        if (error != 0) throw std::bad_alloc();
+        std::unique_ptr<LStrPtr, long (*)(void*)> utf8StringDeleter(utf8String, &DSDisposeHandle);
```

Comment:
Maybe I should swap the throw and the unique_ptr in case there are any error cases that allocate stringOut.

Reply:
Swapped. I also confirmed that DSDisposeHandle allows nullptr.